dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryService.java

```diff
@@ -1,513 +1,523 @@
-package org.hisp.dhis.category;
-
-/*
- * Copyright (c) 2004-2019, University of Oslo
- * All rights reserved.
- *

```

Comment:
Something strange seems to be causing a complete diff on this file.

Reply:
something must have happened with line-ending and git. I switched to linux few days ago...